docs/codeql/codeql-workshops/cpp/codeql-dataflow-sql-injection/codeql-dataflow-sql-injection.md

```diff
@@ -0,0 +1,983 @@
+<!-- -*- coding: utf-8 -*- -->
```

Comment:
There are checked-in `.pdf` and `.key` files in this directory. I would prefer to have those omitted, and keep only the raw Markdown contents.

Reply:
The `.key` files are the source of the `.pdf` files.  Some are diagrams that really don't work as markdown;  they **might** be convertible to multiple SVG pages -- but then there is no way to edit them.